module/solutions/helper.js

```diff
@@ -719,11 +722,14 @@ module.exports = class SolutionsHelper {
           "scope.roles.code" : { $in : [constants.common.ALL_ROLES,data.role] },
           "scope.entities" : { $in : entityIds },
           "scope.entityType" : { $in : entityTypes },
-          isReusable : false,
-          "isDeleted" : false,
-          status : constants.common.ACTIVE
+          "isReusable" : false,
+          "isDeleted" : false
         };
-    
+
+        if(type !==  constants.common.SURVEY){
```

Comment:
@priyanka-TL for survey should we not do a date check of not more than 15 days?

Reply:
@aks30 that we are doing inside the loop. we are fetching solution so how we can check endDate +15 day condition in query